### ISSUE TITLE
[please remove] Simple Storage Network

### ISSUE THREAD

**Issue by happypsion** [kreezxil/World-of-Dragons#21]
# Mods Nominated for Removal
Mod Name:  Simple Storage Network

# Reason for Nomination
Reason:  Redundant

This pack already has Refined Storage, which is augmented with the addons to it that are in the pack:
Refined Storage Addons
Reborn Storage
StorageTech

I see no reason to have another system that does the same thing but isn't as advanced and contributed to.

**Comment by kreezxil** [comment]
Good point, Simple Storage Network simply came before RS which is why it's in the pack. So this one is definitely going.
